server/routes/profileScores.ts

```diff
@@ -37,6 +37,8 @@ function init({ genomes }) {
 }
 
 async function getScores(query, ds, genome) {
+	query.__protected__.ignoredTermIds.push(query.facilityTW.term.id)
```

Comment:
We need to add this line too in profileFormScores that is a similar route that calculates the scores for the template plots